blog/2019-09-ngrx-data-views/README.md

```diff
@@ -0,0 +1,419 @@
+---
+title: "Handling normalized entities with data views"
+author: Danilo Hoffmann
+mail: dhhyi@aol.com
+published: 2019-09-30
+last-change: 2019-09-30
+keywords:
+  - Angular
+  - RxJS
+  - Reactive Programming
+  - Observables
+  - NgRx
+  - NgRx Entities
+  - Memoization
+language: en
+thumbnail: data-views-header.jpg
+hidden: false
+---
+
```

Comment:
Provide summary with problem description in 2-3 sentences. See the other blog posts for examples. The goal is to provide context and tell readers what they can expect. The intro should be in easy language so that people can just read it quickly to see whether the article is good for them.

Reply:
I tried...

Reply:
I tried